signer/sign_state.go

```diff
@@ -107,7 +107,11 @@ func (signState *SignState) GetFromCache(hrs HRSKey, lock *sync.Mutex) (HRSKey,
 	return latestBlock, nil
 }
 
-func (signState *SignState) Save(ssc SignStateConsensus, lock *sync.Mutex, async bool) error {
+func (signState *SignState) Save(
```

Comment:
This method should be documented. Exported methods accepting a mutex and a wait group are very unusual.

Reply:
documented.